src/main/java/HomePage.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HomePage {
+    private final WebDriver driver;
+    private final String EXPECTED_NAME = "Inbox — Yandex.Mail";
```

Comment:
Expected values (test data) must be in test classes

Reply:
This was changed